### PR TITLE
Touch pressure support on ios and android

### PR DESCRIPTION
**Pull request by piotr-j**
Implemented touch pressure on android and ios, modified multitouch test to use it.
Could use some feedback!
#5241

### CODE REVIEW

**Comment by piotr-j** [comment]
Any idea why does the pr check fails?

**Comment by lukz** [comment]
Awesome! Really need this one!

File: backends/gdx-backend-headless/src/com/badlogic/gdx/backends/headless/mock/input/MockInput.java [hunk 1]
```diff
@@ -110,6 +110,14 @@ public boolean isTouched(int pointer) {
 		return false;
 	}
 
+	@Override public float getPressure () {
```
Comment (Tom-Ski) [inline backends/gdx-backend-headless/src/com/badlogic/gdx/backends/headless/mock/input/MockInput.java]: Can you format these overrides?

**Comment by piotr-j** [comment]
Any other dumb things that i missed?

**Comment by balepc** [comment]
This is very expected!!! Could you, please, merge it?

**Pull request merged**